changelog/_unreleased/2021-01-14-set-total-after-fetching-list-items.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Set total after fetching list items
+issue: NEXT-10679
+author: Elias Lackner
+author_email: lackner.elias@gmail.com
+author_github: @lacknere
+---
+# Administration
+*  Setting the value of `total` after fetching list items in `sw-settings-list.mixin.js`
```

Comment:
the change log entry does not match our conventions. have a look here https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md#changelog-entries